extensions-contrib/opentelemetry-metrics-extensions/src/test/java/org/apache/druid/data/input/opentelemetry/protobuf/OpenTelemetryMetricsProtobufReaderTest.java

```diff
@@ -0,0 +1,442 @@
+/*
```

Comment:
1. Could you please add a proto file to the test/resources and try to read it from from there ? It can be a new test case as well . 
2. Add tests cases for unsupported columns/data types